src/freertos_drivers/common/DccDecoder.hxx

```diff
@@ -200,67 +196,27 @@ private:
         MAP_TimerEnable(HW::TIMER_BASE, HW::SAMPLE_TIMER);
         }*/
 
+    /// Module::set_cap_timer_delay_usec(int usec)
```

Comment:
Please verify that this syntax gets picked up by Doxygen to associate the comment with DccDecoder::Module::set_cap_timer_delay_usec().

Reply:
No it isn't picked up. I removed these. These docu comments are in the TivaDccDecoder, and I added a link to point there.